features/specs/S-en-T-test-omgekeerd-ouders/Tg001.feature

```diff
@@ -734,6 +746,18 @@ Functionaliteit: Tg001 - Adema-Arendsen - Gehuwd met kinderen geboren voor en na
       | type                             | EenhoofdigOuderlijkGezag |
       | minderjarige.burgerservicenummer | 000000085                |
       | ouder.burgerservicenummer        | 000000024                |
+    En heeft de persoon een 'gezag' met de volgende gegevens
```

Comment:
deze klopt niet! Als de gezag API dat nu levert is het m.i. een bug. Je vraagt gezag van 000000024 en je krijgt het gezag van 000000012 en een OnbekendeDerde over 000000115 en 000000103. Je mag een gezagsrelatie alleen leveren waarbij de bevraagde persoon minderjarige, ouder of derde is in dat gezag.